az-core/src/main/java/azkaban/Constants.java

```diff
@@ -301,6 +301,9 @@
     public static final String PROJECT_CACHE_SIZE_PERCENTAGE = "azkaban"
         + ".project_cache_size_percentage_of_disk";
 
+    public static final String PROJECT_CACHE_THROTTLE_PERCENTAGE = "azkaban"
+        + ".project_cache_throttle_percentage";
```

Comment:
Please consider keeping the config key in one line as it will help in case someone is looking at how this config is used.
public static final String PROJECT_CACHE_THROTTLE_PERCENTAGE = 
        "azkaban.project_cache_throttle_percentage";